bluepysnap/config.py

```diff
@@ -23,6 +23,19 @@
 from bluepysnap import utils
 from bluepysnap.exceptions import BluepySnapError
 
+EXPECTED_PATH_KEYS = ['morphologies_dir',
```

Comment:
Why hardcoding this? Do we always expect relative paths for them? What if an absolute path comes under these keys?

Reply:
No, this is to momentarily solve the issue of how to handle relative filenames that are not defined with leading `./`. Momentarily meaning whenever libsonata has config "parser" for simulations (they already have for circuits) I will deprecate the whole `bluepysnap.config` submodule.

Reply:
That is great to hear. If `libsonata` can be reused then it is great. Ok to have it as temporary.

Reply:
They are the list of keys that we expect to have paths (in order to resolve relative paths that don't begin with `./`). I added a comment to explain this.

Reply:
No particular reason other than preference. Set makes sense here, so I might as well use that.